app.sh

```diff
@@ -4,20 +4,32 @@
 # command and debug level
 #
 
-if [ "$SUBCOMMAND" = "producer" ]
+set -o nounset
+set -o errexit
+set -o errtrace
+set -o pipefail
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+
+# TODO(pacospace) maybe this is a good inspiration: https://github.com/xwmx/bash-boilerplate/blob/master/bash-commands
+
+FAUST_COMMAND="faust --web-host 0.0.0.0"
```

Comment:
Why do we set the `web-host` option?

Reply:
So that it binds to all Interfaces and not just localhost